mw-webapp/src/model/businessModel/JobDone.ts

```diff
@@ -16,21 +16,27 @@ export class JobDone {
   public description: string;
 
   /**
-   * How long was the job done
+   * Enum @Time.unit (minute, hour, day, etc.)
    */
-  public time: Time;
+  public timeUnit: string;
 
-  constructor(jobDoneData: JobDone) {
-    this.uuid = jobDoneData.uuid;
-    this.description = jobDoneData.description;
-    this.time = new Time(jobDoneData.time.unit, jobDoneData.time.amount);
+  /**
+    * How long did the task take to complete
```

Comment:
Do we have `task` in business models?  If no - what is task

Reply:
Rename task to job